packages/ui/src/components/contact/Contact.tsx

```diff
@@ -0,0 +1,46 @@
+"use client"
+import { Mail, MapPin, Phone } from "lucide-react"
+
+export interface ContactProps {
+  phone?: string
+  email?: string
+  address?: string
+  title?: string
+  className?: string
+}
+
+export function Contact({
+  address = "Merkez Mah. Destek Cad. No:1, İstanbul",
+  className,
+  email = "info@lotusvdmedia.nl",
+  phone = "+90 (212) 555 0000",
+  title = "Bize Ulaşın",
+}: ContactProps) {
```

Comment:
Don't use initial props, could you provide these props from in page where you use the ContactComponent and also in the story?